src/reducer/quizReducer.js

```diff
@@ -37,16 +37,23 @@ export const quizReducer = (quizState, { type, payload }) => {
         score:
           payload === quizState.questions?.[quizState.currentQuestion]?.correctAnswer
             ? quizState.score + 10
-            : quizState.score - 10
+            : quizState.score
       };
```

Comment:
I couldn't understand the reason for using optional chaining here, please share your thoughts so i can learn as well.

Reply:
done this for fail-safe purpose, initially questions is an empty array, in empty array current question will not be available that would result in an error hence I did this so it doesn't return an error